File: Calculate First 1000 Primes/JS/ThousandPrimes.js
let isPrime = (number) => {
  let check = [2, 3, 5, 7, 11];

  for (let i = 0; i < check.length; i++) {
    const x = check[i];
    if (number % x == 0) {
      return false;
    }
  }

  return true;
};

let primes = [2, 3, 5, 7, 11];

let i = 12;
while (primes.length < 1000) {
  if (isPrime(i)) {
    primes.push(i);
  }

  i += 1;
}

primes.forEach((prime) => {
  console.log(prime);
});

File: Calculate First 1000 Primes/Python/thousand_primes.py
def is_prime(number):
    check = [2, 3, 5, 7, 11]

    for x in check:
        if number % x == 0:
            return False

    return True


primes: list = [2, 3, 5, 7, 11]
i: int = 12
while len(primes) < 1000:
    if is_prime(i):
        primes.append(i)
    i += 1

for prime in primes:
    print(prime)
